frontend/src/components/static_pages/contact.css

```diff
@@ -0,0 +1,84 @@
+
+*{
+    margin:0;
+    padding:0;
+  }
+  
+  .messageInput {
+      width: 40%;
+  }
+  
+  .jumbotron {
+      margin: auto 0;
+      justify-content: left;
+  }
+  
+  span{
+      justify-content: center;
+  }
+  
+  .main-content {
+      margin: auto;
+  }
+  
+  
+  .block{
+      display:block;
```

Comment:
great thinking here. but we can simplify it bit. 

you can just do this on the `label` and remove the `block` class from the html

```
label {
  display: block;
}
```